app/src/test/java/com/kickstarter/UserCurrencyTest.kt

```diff
@@ -0,0 +1,76 @@
+package com.kickstarter
+
+import com.kickstarter.libs.UserCurrency
+import com.kickstarter.mock.MockCurrentConfig
+import com.kickstarter.mock.factories.ConfigFactory
+import com.kickstarter.mock.factories.ProjectFactory
+import junit.framework.Assert
+import junit.framework.TestCase
+import type.CurrencyCode
+import java.math.RoundingMode
+
+class UserCurrencyTest: TestCase() {
+
+    private fun createUserCurrency(countryCode: String): UserCurrency {
+        val config = ConfigFactory.config().toBuilder()
+                .countryCode(countryCode)
+                .build()
+
+        val currentConfig = MockCurrentConfig()
+        currentConfig.config(config)
+
+        return UserCurrency(currentConfig)
+    }
+
+    fun testFormatCurrency_withUserInUS() {
```

Comment:
I'm not sure what these tests are testing. Is it testing that the user prefers a currency or they're in the a locale? Let's rename it to what it's actually testing.

Reply:
So these tests are testing where the user is and how the currency should be displayed based on the locale.